install_config/configuring_local.adoc

```diff
@@ -148,7 +148,7 @@ ifdef::openshift-origin[]
   -p PROVISIONER_IMAGE=quay.io/external_storage/local-volume-provisioner:v1.0.1 \
 endif::[]
 ifndef::openshift-origin[]
-  -p PROVISIONER_IMAGE=registry.access.redhat.com/openshift3/local-storage-provisioner:v3.9 \ <1>
+  -p PROVISIONER_IMAGE=registry.access.redhat.com/openshift3/local-storage-provisioner:v3.10 \ <1>
```

Comment:
This update is not necessary.

Reply:
True. I only wanted the document to be consistent.